libcurl.def

```diff
@@ -91,4 +92,4 @@ curl_version
 curl_version_info
 curl_ws_meta
 curl_ws_recv
-curl_ws_send
+curl_ws_send
```

Comment:
I suggest keeping the ending linefeed.

Reply:
Yeah, it was linefeed ending missing, indeed! Thanks a lot!
The remaining test failures don't seem to be caused by my changes.